README.md

```diff
@@ -67,6 +67,30 @@ It supports the following options:
 There's a simple vscode extension available for download here: [download vix](https://github.com/pherrymason/c3-lsp/releases/download/latest/c3-lsp-client-0.0.1.vsix)  
 Be sure to configure it with the path of the lsp server binary.
 
+### Sublime Text
+
+The [LSP Package](https://packagecontrol.io/packages/LSP) can be configured to
+work with `c3-lsp`. Add the [syntax highlighting package](https://github.com/c3lang/editor-plugins/tree/main/sublime-text) and configure a LSP client in `Settings` -> `Package Settings` -> `LSP` -> `Settings` like the following:
+
+```yaml
+"clients": {
+    "c3-lsp": {
+      "enabled": true,
+      // The command line required to run the server.
+      "command": [
+        "c3-lsp",
+      ],
+
+      "selector": "source.c3",
```

Comment:
there are also files with `c3l` extension. Should it be added here too?

Reply:
No, this is set by the syntax package https://github.com/c3lang/editor-plugins/tree/main/sublime-text, that's why this package needs to be installed too. This is like VS Code's language definition.

Reply:
But looking at https://github.com/c3lang/editor-plugins/blob/main/sublime-text/c3.sublime-syntax#L4 I only added the extensions `c3` und `c3t`. I'll add `c3l` there too, I guess?

Reply:
I don't know about `c3t`, but `c3l` was needed for visual studio extension.

Reply:
It's `c3i` (correct in VS Code's package.json) and the `c3t` is somehow a typo :).